torch/_logging/_registrations.py

```diff
@@ -2,7 +2,11 @@
 from ._internal import register_artifact, register_log
 
 DYNAMIC = ["torch.fx.experimental.symbolic_shapes", "torch.fx.experimental.sym_node"]
-DISTRIBUTED = ["torch.distributed", "torch._dynamo.backends.distributed"]
+DISTRIBUTED = [
+    "torch.distributed",
+    "torch._dynamo.backends.distributed",
+    "torch.nn.parallel.distributed",
```

Comment:
missing fsdp here?

Reply:
Oh, fsdp is a child of "torch.distributed", so it should be in already.